dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -56,7 +156,92 @@ public SauceOptions()
         public string TimeZone { get; set; }
         public string TunnelIdentifier { get; set; }
         public bool VideoUploadOnPass { get; set; }
+        public DriverOptions SeleniumOptions { get; set; }
+        public Browser BrowserName { get; set; } = Browser.Chrome;
+        public static List<string> w3cOptions = new List<string>(new string[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)